### DISCUSSION TITLE
abctl local commands: start, stop, restart

### DISCUSSION THREAD

**Discussion by erlangparasu** [airbytehq/airbyte#45847] [General]
```
Usage: abctl <command> [flags]

Airbyte's command line tool for managing a local Airbyte installation.

Flags:
  -h, --help       Show context-sensitive help.
  -v, --verbose    Enable verbose output.

Commands:
  local credentials [flags]
    Get local Airbyte user credentials.

  local install [flags]
    Install local Airbyte.

  local deployments [flags]
    View local Airbyte deployments.

  local status
    Get local Airbyte status.

  local uninstall [flags]
    Uninstall local Airbyte.

  version [flags]
    Display version information.

```

There are `abctl local install`, `abctl local uninstall`

Could we have `abctl local stop`, `abctl local start`, `abctl local restart` ?

**Comment by abuchanan-airbyte** [comment]
Could you describe what you would expect these commands to do?

**Comment by marcosmarxm** [comment]
From me:
* `stop`: stop the kind cluster to release resources in my local machine. Today `uninstall` kind do this but as an user it isn't clear to use `uninstall` command to stop.
* `restart` maybe is for people who applied a change to the `values.yaml`, we can use `install` but from the help isn't clear.

**Reply by abuchanan-airbyte** [reply]
It's possible to start/stop easily today with `docker start/stop airbyte-abctl-control-plane`

**Reply by cbuckle1** [reply]
Tried using docker stop airbyte-abctl-control-plane and then docker start airbyte-abctl-control-plane; didn't work gave a 404 error. Ended up using local uninstall and local install to get it back up

**Reply by aaronsteers** [reply]
Agreed. I'd love to have `start` and `stop`. On another thread, I proposed these as `up` and `down`, e.g.:

- `abctl local up` would bring Airbyte up (starting it) if it isn't already up.
- `abctl local down` would bring Airbyte down (stopping it) if it isn't already down.

I don't feel strongly on what they are called - I like `start` and `stop` just as well.

**Reply by scottmorgan-payroc** [reply]
docker stop airbyte-abctl-control-plane    
blows away the container

**Comment by scottmorgan-payroc** [comment]
In my experience

1. to restart use   **_docker restart airbyte-abctl-control-plane_**
2. to blow away the container _**docker stop airbyte-abctl-control-plane**_
